types/react-native/index.d.ts

```diff
@@ -25,6 +25,7 @@
 //                 Xianming Zhong <https://github.com/chinesedfan>
 //                 Valentyn Tolochko <https://github.com/vtolochk>
 //                 Sergey Sychev <https://github.com/SychevSP>
+//                 Kelvin Chu <https://github.com/RageBill>
```

Comment:
Why are you adding this owner?

Reply:
This was a copy from `react-native/types/index.d.ts` -- I guess this person added themselves to that one? Will remove